source/features/reactions-avatars.tsx

```diff
@@ -62,7 +62,7 @@ async function showAvatarsOn(commentReactions: Element): Promise<void> {
 		container.append(
 			// Without this, Firefox will follow the link instead of submitting the reaction button
 			<a href={isFirefox ? undefined : `/${username}`} className="rounded-1 avatar-user">
-				<img src={imageUrl} style={{borderRadius: 'inherit'}}/>
+				<img src={imageUrl} className="rounded-1"/>
```

Comment:
You should use `className="avatar-user rounded-1"` becasue `rounded-1` is for old design and has smaller radius:
![image](https://user-images.githubusercontent.com/44045911/86535285-d78e5500-bf11-11ea-9389-3982de0ce201.png)

Reply:
```suggestion
				<img src={imageUrl} className="avatar-user rounded-1"/>
```

Reply:
@yakov116 Why?

Reply:
Followed @kidonng https://github.com/sindresorhus/refined-github/pull/3314#discussion_r449885938
> You should use `className="avatar-user rounded-1"` because `rounded-1` is for old design and has smaller radius (`6px`):